config/samples/kustomization.yaml

```diff
@@ -2,5 +2,6 @@
 resources:
 - toolchain.dev.openshift.com_v1alpha1_usersignup.yaml
 - toolchain.dev.openshift.com_v1alpha1_space.yaml
+- toolchain.dev.openshift.com_v1alpha1_socialevent.yaml
```

Comment:
that came from operator-sdk, but I'm not sure we need this change

Reply:
removed in https://github.com/codeready-toolchain/host-operator/commit/8a32d924810cca8098cff6c76cf003c9bf2153db